src/main/scala/io/iohk/ethereum/jsonrpc/TransactionResponse.scala

```diff
@@ -14,13 +14,15 @@ case class TransactionResponse(
     value: BigInt,
     gasPrice: BigInt,
     gas: BigInt,
-    input: ByteString)
+    input: ByteString,
+    pending: Option[Boolean])
```

Comment:
What's the difference between what `None` and `Some(false)` represent? Maybe it is enough to have a boolean representing whether the tx is pending or not?

Reply:
I made it an Option, so that we don't include this flag in other endpoints (and so that they match spec)

Reply:
Maybe it would be better then to return a `Some(false)` when the tx was obtained from a block in [`eth_getAccountRecentTransactions`](https://github.com/input-output-hk/mantis/pull/330/files#diff-4aa60425d2cdec6299a3815b98a8e30dR750) instead?

Reply:
Yes, I missed that, good catch